src/django_components/slots.py

```diff
@@ -124,7 +124,6 @@ class SlotFill(Generic[TSlotData]):
     escaped_name: str
     is_filled: bool
     content_func: SlotFunc[TSlotData]
-    context_data: Mapping
```

Comment:
`context_data` was wired all the way to inside `SlotNode.render()`, but it wasn't actually used, so I removed it.